internal/nullable/value.go

```diff
@@ -16,10 +16,17 @@ type Value[T any] struct {
 	Value T
 }
 
-// FromString[T any] represents a parser function. It is used to avoid
+func New[T nullable](v T, set bool) Value[T] {
```

Comment:
If we have `nullable`, then `value` could not be null 
example:
`var a int = 0`
Then the following is not true
`New(a,false)`
a is not null, I see `Set` any value or `nil` by default `int` goes to `0` which is not `nil`
So new function can be like `New(a)`

If some one need to have int with null, then we can provide `NewNull[int]()`
`a := NewNull[int]()`